addons/microsoft_calendar/models/calendar.py

```diff
@@ -163,6 +194,15 @@ def write(self, values):
             self.recurrence_id.need_sync_m = True
         return res
 
+    def _get_organizer_user_change_info(self, values):
+        """ Return the sender user of the event and the partner ids listed on the event draft. """
+        sender_user_id = values['user_id'] if 'user_id' in values else self.env.user.id
+        sender_user = self.env['res.users'].search([('id', '=', sender_user_id)])
+        param_partner_ids = self._attendees_values(values['partner_ids']) if 'partner_ids' in values else []
```

Comment:
Note that when you are coming from a write, you could be deleting attendees and they will be in the param_partner_ids, so you would need to check command by command.

Reply:
> know if this could happen inside the write() function because, for instance, removing a partner 'A' would also remove 'A' from the parameter 'partner_ids' in values.

Most times yes, because when you are writing from the UI the request sends the `partner_ids` with a Set command, but there could be cases (e.g. in tests) where you write on `partner_ids` with other commands like unlink. Not sure if we should adapt this code to handle these cases because it's a lot of complexity. What do you think @arj-odoo?

Reply:
yes, it could be quite complex ad fragile. Let's try it like that for now.